Set-DomainUser.ps1

```diff
@@ -752,12 +896,173 @@ Function Activate-Platform {
+            specifyNextAccountContent              = $False
+            renameAccounts                         = $False
+            deleteAccounts                         = $False
+            unlockAccounts                         = $False
+            manageSafe                             = $False
+            manageSafeMembers                      = $False
+            backupSafe                             = $False
+            viewAuditLog                           = $False
+            viewSafeMembers                        = $False
+            accessWithoutConfirmation              = $False
+            createFolders                          = $False
+            deleteFolders                          = $False
+            moveAccountsAndFolders                 = $False
+            requestsAuthorizationLevel1            = $False 
+            requestsAuthorizationLevel2            = $False
+        }
+        New-SafePermissions -pvwaAddress $pvwaAddress -pvwaToken $pvwaToken -safe $safe -safeMember "PVWAAppUser" -memberType "user" -safePermissions $PVWAAppUser
+        New-SafePermissions -pvwaAddress $pvwaAddress -pvwaToken $pvwaToken -safe $safe -safeMember "PVWAAppUsers" -safePermissions $PVWAAppUser
+        #PSMAppUsers
+        $PSMAppUsers = @{
+            useAccounts                            = $False 
+            retrieveAccounts                       = $True
+            listAccounts                           = $True
+            addAccounts                            = $False 
+            updateAccountContent                   = $False
+            updateAccountProperties                = $False
+            initiateCPMAccountManagementOperations = $False
+            specifyNextAccountContent              = $False
+            renameAccounts                         = $False
+            deleteAccounts                         = $False
+            unlockAccounts                         = $False
+            manageSafe                             = $False
+            manageSafeMembers                      = $False
+            backupSafe                             = $False
+            viewAuditLog                           = $False
+            viewSafeMembers                        = $False
+            accessWithoutConfirmation              = $False
+            createFolders                          = $False
+            deleteFolders                          = $False
+            moveAccountsAndFolders                 = $False
+            requestsAuthorizationLevel1            = $False 
+            requestsAuthorizationLevel2            = $False
+        }
+        Set-SafePermissions -pvwaAddress $pvwaAddress -pvwaToken $pvwaToken -safe $safe -safeMember "PSMAppUsers" -safePermissions $PSMAppUsers  
+        return $true      
+    }
+    catch {
+        Write-LogMessage -Type Error $_.ErrorDetails.Message
+        return $false
+    }
+}
+
+Function Set-SafePermissions {
```

Comment:
Why did I make a set and a new function? I think set should only be if the permissions are already there?

Reply:
I added the new function